testsuite/integration-arquillian/tests/base/src/test/resources/arquillian.xml

```diff
@@ -36,13 +36,19 @@
         <property name="firefoxBinary">${firefox_binary}</property> <!-- we need to use 'firefoxBinary' instead of 'firefox_binary' due to some weird conflict with Appium -->
         <property name="firefoxLogLevel">OFF</property>
         <property name="firefoxLegacy">${firefoxLegacyDriver}</property>
+        <property name="firefoxDriverBinary">${firefoxDriverBinary}</property>
```

Comment:
Why to add another properties for path to driver binary? Drone has already some built-in ones, see https://github.com/keycloak/keycloak/blob/master/testsuite/integration-arquillian/HOW-TO-RUN.md#automatic-driver-downloads

Reply:
To support both ways of setting path to driver binary: one via `webdriver.*.driver`(already supported) the other via `*DriverBinary`see properties table in: http://arquillian.org/arquillian-extension-drone/#_how_it_works

Reply:
But I guess it is not necessary and perhaps it would be clearer it there was only one way supported by the testsuite? Also it's not directly related to the issue.

Reply:
> But I guess it is not necessary and perhaps it would be clearer it there was only one way supported by the testsuite? Also it's not directly related to the issue.

+1